.circleci/config.yml

```diff
@@ -96,12 +105,11 @@ jobs:
           name: Apply Overrides
           command: make init_multisig
           working_directory: ~/repo/MultiSigWalletOverride
+      # node symlink missing if there's not nodejs-legacy
+      - run: sudo apt-get update && sudo apt-get install -y build-essential libhidapi-dev libudev-dev libusb-1.0-0 libusb-1.0-0-dev nodejs-legacy
```

Comment:
what's happening to the original nvm installation?

Reply:
the intial problem was that ubuntu 14 (which is what cirleci uses if you don't define an image) uses a really old gcloud and docker and docker compose. that didn't allow me to authenticate towards our GCS registry. so I had to bump to ubuntu 16. `image: ubuntu-1604:201903-01`
now that broke helluva lot of things with gnosis 1.6 tag so I had to install `nodejs-legacy` to even be able to `npm install`.